### ISSUE TITLE
Read Rainbow 5.37.6 file with different number of bins between sweeps

### ISSUE THREAD

**Issue by pedroaugustosmribeiro** [ARM-DOE/pyart#952]
Please check if it's possible to parse a RAINBOW file version 5.37.6 that gives the error: "Number of ranges bins changes between sweeps"
It's readable on wradlib.

**Comment by zssherman** [comment]
@pedroaugustosmribeiro Yeah, might need to add a max bin and mask bins added to lower bin sweeps for sweeps with differing shapes. I believe something similar to this exists in one of the nexrad readers, so I'll see if I can track it down.

**Comment by tsbiscaro** [comment]
Any update on this issue?

**Comment by zssherman** [comment]
Sorry for the very late response. Our goal is eventually to use xradar as the io portion of reading radar files into xarray. I believe:
https://docs.openradarscience.org/projects/xradar/en/stable/notebooks/Rainbow.html
should handle different number of bins between sweeps.
Then can use it within pyart as such:
https://arm-doe.github.io/pyart/examples/xradar/plot_xradar.html#sphx-glr-examples-xradar-plot-xradar-py

**Comment by zssherman** [comment]
Moving users to use xradar rainbow reader, closing.
